modules/ossm-observability-access.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+//- ossm-observability.adoc
+
+[id="ossm-observability-access-console_{context}"]
+= Accessing the Kiali console
+
+To access the console, in the menu bar, click the *Application launcher* > *Kiali*.
```

Comment:
I'm just thinking, it's entirely possible that someone might be given a direct link to the Kiali Console.  So maybe we should add some context for where we're starting from and say "... in the menu bar of the OpenShift console, click...."

Reply:
made a change

Reply:
I think I liked it better before when the lead in was just about accessing the Kiali console.  =(  I'd kind of like to avoid the word "observability" where we can, if only because it's so jargon-y and I'm not sure how it translates.  (I just noticed it's in the file name).